test/lazy-loading.html

```diff
@@ -925,6 +925,38 @@
               expect(comboBox.value).to.equal('item 12');
             });
 
+            it('should set value without auto-open-disabled even if case does not match', () => {
+              comboBox.autoOpenDisabled = false;
+              expect(comboBox.autoOpenDisabled).to.be.false;
+
+              comboBox._inputElementValue = 'ItEm 12';
+              comboBox.filter = 'item 12';
```

Comment:
To help me understand this: `filter` is the value that the last data provider query was made for, and `_inputElementValue` is the current state of the input?

Reply:
Thanks for pointing it out. This was wrong. They both should have the same value, as `filter` gets the value of `_inputElementValue` on the input event.